python/ray/actor.py

```diff
@@ -208,7 +209,7 @@ def __init__(self, modified_class, class_id, max_reconstructions, num_cpus,
         self._num_cpus = num_cpus
         self._num_gpus = num_gpus
         self._resources = resources
-        self._exported = False
+        self._last_export_session = None
```

Comment:
We initialize this to `None` instead of to `global_worker._session_index` because we don't actually export the actor class here.